ipol_demo/clientApp/demo.html

```diff
@@ -119,8 +126,14 @@ <h1 class="container-title m-y-5">Results</h1>
   <script src="js/demo.results.js"></script>
   <script src="js/demo.results.gallery.js"></script>
   <script src="js/demo.results.video-gallery.js"></script>
+  <script src="js/demo.map.js"></script>
 
   <script src="https://cdnjs.cloudflare.com/ajax/libs/cropper/3.0.0/cropper.min.js" defer></script>
 
+  <script src='https://api.mapbox.com/mapbox-gl-js/v2.9.1/mapbox-gl.js'></script>
+  <link href='https://api.mapbox.com/mapbox-gl-js/v2.9.1/mapbox-gl.css' rel='stylesheet' />
+  <script src="https://unpkg.com/@turf/turf@6/turf.min.js"></script>
+  <script src="https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-draw/v1.4.0/mapbox-gl-draw.js"></script>
+  <link rel="stylesheet" href="https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-draw/v1.4.0/mapbox-gl-draw.css" type="text/css">
```

Comment:
Mapbox is not opensource, and additionally requires a token. I suggest we use https://maplibre.org/ instead, it should be mostly compatible with mapbox API.

Reply:
For the moment we'll proceed with mapbox.